src/AffectedUserFlow/exposureNotificationAPI.ts

```diff
@@ -40,7 +57,7 @@ export const postDiagnosisKeys = async (
   hmacKey: string,
   appPackageName: string,
   revisionToken: string,
-): Promise<NetworkResponse<PostKeysSuccess, PostKeysError>> => {
```

Comment:
Why are we moving away from the network response either type? do we not what to make a distinction between a success or error case?

Reply:
My reasoning is behind that a `NetworkResponse` implies that the component is interacting with the network layer, which we are abstracting here using the helper. Having it respond with the result of a network call hits me of as the component interacting with the network which shouldn't be? There is still a distinction between error and success, the error nature of the response includes that.